README-JP.md

```
@@ -0,0 +1,104 @@
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/Dev-Dami/zario?label=build)](https://github.com/Dev-Dami/zario/actions)
+
+[English](README.md)
+
+
+<br/>
+
+**高速** • **軽量** • **依存関係ゼロ** • **TypeScriptネイティブ**
+
+<br/>
+
+[📖 ドキュメント](#-documentation) · [⚡ クイックスタート](#-quick-start) · [✨ 特徴](#-features) · [🤝 コントリビューション](#-contributing)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)